presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -533,4 +565,37 @@ public Boolean process(Node node, @Nullable Void context)
             return super.process(node, context);
         }
     }
+
+    /**
+     * Verifies that no query node output columns are referenced from ORDER BY expressions.
+     * Only FROM columns are allowed within ORDER BY expressions.
+     */
+    private class VerifyOrderByAggregationVisitor
```

Comment:
what about lambda? If `identifier` is in lambda then it can overload and reuse the same name.